src/main/java/uk/gov/crowncommercial/dts/scale/cat/service/ProcurementEventService.java

```diff
@@ -1218,4 +1285,22 @@ private List<AttachmentInfo> getAttachmentInfoForParameter(final Parameter param
             .secureToken(value.getSecureToken()).build())
         .collect(Collectors.toList());
   }
+
+  /**
+   * Calls RPA to Open Envelope
+   *
+   * @param userEmail
+   * @param password
+   * @param externalReferenceId
+   * @return
+   */
+  public void  callOpenEnvelopeAndUpdateSupplier(final String userEmail, final String password,
+                                                  final String externalReferenceId, final RPAProcessInput processInput) {
+    log.info("Calling OpenEnvelope for {}", externalReferenceId);
+    // Creating RPA process input string
+    var inputBuilder = RPAProcessInput.builder().userName(userEmail).password(password)
+            .ittCode(externalReferenceId);
+    rpaGenericService.callRPAMessageAPI(inputBuilder.build(), RPAProcessNameEnum.OPEN_ENVELOPE);
+
```

Comment:
And please remove extra spaces if possible

Reply:
done